Plotting.m initial commit from J-Rush's fork

J-Rush opened the pull request:
Not exactly sure how pull requests work but I'm pretty sure this is the intended function.

J-Rush committed:
Initial commit of plotting.m. Note that this file is still a WIP and was added at the behest of Mr.
Marshall

marshallpt commented:
This looks great! There's a lot of stuff in here I didn't know existed. Do you plan on adding to the `readme.md` as well so it's easy to find on the repo page without diving into the code? All I did for going for converting `basics.m` to the readme was a lot of copy and pasting, basically. You wrote really good extensive comments already that should be perfect.

J-Rush committed:
Added summary of lines 30-67 from plotting.m to README.md

Summarized and slightly modified example code and comments from lines 30 through 67 from plotting.m to README.md per Patrick's request.

J-Rush committed:
Delete README.html

This file was added erroneously as a consequence of J-Rush's ineptitude.
